translations/strings.yaml

```diff
@@ -64,21 +58,155 @@ settingsDev:
+  SupportWorkflow:
+    # Below is the text for a nav link the user can use to contact support
+    navListItem: Contact support
+    fieldCategoryHeader: Enter your email
+    emailFieldPlaceholder: Email address
+    confirmEmailPlaceholder: Confirm email address
+    fieldCategoryHeader: How can we help you with Mozilla VPN?
+    dropdownLabel: Choose a category
+    subjectFieldPlaceholder: Subject (optional)
+    issueFieldPlaceholder: Describe issue
+    disclaimerText: When you submit, Mozilla VPN will collect technical and interaction data with your email to help our support team understand your issue.
+    privacyNoticeLinkText: Mozilla VPN Privacy Notice
+    primaryButtonText: Submit
+    secondaryActionText: Cancel 
+    # below are options for users to select what type of issue they‘re having
+    dropdownListItem: Payment and billing
+    dropdownListItem: Account issues
+    dropdownListItem: Technical issues
+    dropdownListItem: Request features
+    dropdownListItem: Other
+    responseHeader: Thank you!
+    # %1 should be replaced with the email used to complete the form
+    responseBody: Our support team will contact you at %1 once we’ve been able to look at your issue.
+    responseButton: Done
+    # error state
+    errorHeader: Error submitting your support request
+    errorText: An unexpected error has occurred, please try again.
+    errorButton: Try Again
+
+# v2.5 What’s New? Release notes
+whatsNewReleaseNotes:
+  # tour modal alert on system update
+  tourModalHeader: What’s new in VPN?
+  tourModalBodyText: We’ve added a ton of cool features in this latest release. Take the tour to get a deeper dive into what we’ve added for you!
+  tourModalButtonText: Take the tour
+  multiModalHeader: Multi-hop VPN
+  multiModalBodyText: Multi-hop VPN will route your traffic through a second server for added protection. You can find this feature on the “Select location” screen.
+  supportModalButtonText: Next
+  supportModalHeader: In-app support form
+  supportModalBodyText: The in-app support form will allow you to contact support from within the VPN app. You can find it in the “Get help” section.
+  dnsModalHeader: Custom DNS
+  dnsModalBodyText: Custom DNS servers allow for faster speed using local networks, features like ad-blocking and anti-tracking. You can find this feature in the “Network settings” section.
+  dnsModalButtonText: Done
+# feature tour within settings
+  tourPageHeader: What’s new
+  tourSubHeader: Take the tour
+  multiListItemHeader: Multi-hop VPN
+  multiListItemBody: Multi-hop VPN uses multiple VPN servers instead of one for extra security and privacy.
+  supportListItemHeader: In-app support form
+  supportListItemBody: The in-app support form will streamline the process of getting your issues resolved.
+  dnsListItemHeader: Custom DNS
+  dnsListItemBody: Custom DNS settings allow for faster connection using local servers.
```

Comment:
They can also be non local servers. And it's not just for speed... it's just a custom DNS :/